packages/fuels-signers/src/lib.rs

```diff
@@ -1,11 +1,9 @@
 pub mod provider;
-pub mod signature;
 pub mod util;
 pub mod wallet;
 
```

Comment:
I think we should re-export fuel-crypto here so that users of the SDK don't need to take a separate dependency on fuel-crypto in order to work with Signature and key types.

```suggestion
pub use fuel_crypto;
```